programs/multicall-handler/src/lib.rs

```diff
@@ -12,9 +12,9 @@ declare_id!("6kqWTz3A3ZYMV2FMU24ke8rHzT82SaBz7GkBKTd7Z9BH");
 pub mod multicall_handler {
     use super::*;
 
-    // Handler to receive AcrossV3 message formatted as serialized message compiled instructions. When deserialized,
+    // Handler to receive Across message formatted as serialized message compiled instructions. When deserialized,
     // these are matched with the passed accounts and executed as CPIs.
-    pub fn handle_v3_across_message(ctx: Context<HandleV3AcrossMessage>, message: Vec<u8>) -> Result<()> {
+    pub fn handle_across_message(ctx: Context<HandleAcrossMessage>, message: Vec<u8>) -> Result<()> {
```

Comment:
This would require updating instruction discriminator used in Spoke [here](https://github.com/across-protocol/contracts/blob/446b5a185682949556f391cf28d2a1e43a706cfe/programs/svm-spoke/src/utils/message_utils.rs#L8-L9)

Reply:
ok then let's leave it with the v3 in it.